src/components/Inputs/TextInput/TextInput.js

```diff
@@ -54,26 +48,28 @@ function PrivateTextInput({
 
   // Set up validation hooks
   const [getError, setError, validate] = useErrorMessage(validator)
-  const [minMaxValidator] = useMinMaxLength(minLength, maxLength)
 
   const [value, setValue] = useState('')
 
-  const doValidation = (value) => {
-    const minMaxError = minMaxValidator(value)
-    if (minMaxError) {
-      setError(minMaxError)
-      return
+  const setErrorWrapper = (errorValue) => {
+    if (!!formChangeHandler) {
```

Comment:
Is formChangeHandler leftover? I don't see in proptypes or example. If not, can you add those please?

Reply:
didn't do proptypes for these yet because this is a prototype. formChangeHandler gets injected here: https://github.com/getethos/ethos-design-system/pull/22/files#diff-c17788385171ff0dfafebf350fa8aa26R51

if that makes sense, please resolve this conversation

Reply:
it gets inserted here: https://github.com/getethos/ethos-design-system/pull/22/files#diff-c17788385171ff0dfafebf350fa8aa26R51